integrationTests/vm/txsFee/relayedScCalls_test.go

```diff
@@ -11,14 +11,17 @@ import (
 	"testing"
 
 	"github.com/multiversx/mx-chain-go/config"
+	"github.com/multiversx/mx-chain-go/integrationTests"
```

Comment:
please add a semiintegration test here in which you test the transaction fee is computed correctly if the DynamicGasCostForDataTrieStorageLoad is activated. Suggestion to have 2 tests: one when using the SaveKeyValue builtin function and one using a contract that can add data (maybe using an internal array or something).

Reply:
Added.